data/lightcode.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "LightCode",
+    "url": "https://lightcode.net/",
+    "career_page_url": "https://lightcode.net/",
+    "type": "Consulting",
+    "categories": [
+       "cloud_software"
+     ],
+     "remote_policy": "Full",
```

Comment:
@omid-ehsani any chance you can point us towards some evidence about full remote employment? Couldn't find any. Even a direct experience is good enough

Reply:
that would be awesome, thanks!

Reply:
Here are some of our remote workers, they will attest LightCode's full-remote collaboration model:
@DiegoSilvaLightCode @eliodimauro @giovanni-lisitano-lc @luca-lc-colombo @mirko-pozzetti-lightcode @nazarenom @oleg-lc @PasqualeIapalucci @serpratt @umbertocollina-lightcode

Reply:
Hello, I'm working full remote from Friuli Venezia Giulia for LightCode

Reply:
Hi all, I'm working full remote from Calabria for LightCode.

Reply:
Here I am! I do all my work remotely from the province of Milan.

Reply:
Hi, I'm working full remote from province of Milan for Lightcode

Reply:
Hi, I'm working full remote for LightCode.

Reply:
Here I am! I do all my work remotely from the province of Latina, Terracina, for Lightcode.

Reply:
@edoardocostantinidev do you think these referrals satisfy the requirements? Thx.

Reply:
super ! thanks everyone

Reply:
Hi, I'm also are a remote co-worker of LightCode.